packages/ERTP/src/paymentLedger.js

```diff
@@ -20,7 +20,7 @@ const { details: X, quote: q, Fail } = assert;
 /**
  * @param {Brand} brand
  * @param {AssetKind} assetKind
- * @param {Matcher} elementShape
+ * @param {Pattern} elementShape
```

Comment:
The original should always have been `Pattern` rather than `Matcher`. Good that hiding the `Matcher` type reveals these. For most purposes, the `Matcher` type is a detail within the meaning of the `Pattern` type.